### ISSUE TITLE
Add to F-Droid

### ISSUE THREAD

**Issue by pizzamaker** [joschi70/AndroidPirateBox#20]
It would be handy to have this app distributed through F-Droid. Do you think that would be feasible?

**Comment by qwertygc** [comment]
You can ask to F-Droid Team, they choose app on repo :-)

**Comment by anto1ne** [comment]
I am going to do a request on f-droid.org.

edit : it is [here](https://gitlab.com/fdroid/rfp/issues/899)

**Comment by licaon-kter** [comment]
What's up with the `libs` folder?

F-Droid needs to either build those from source or get them from a trusted maven repo.

**Comment by anto1ne** [comment]
Here are some updates 
https://gitlab.com/fdroid/rfp/issues/899#note_141213544

> And the tag name (0.5.9) doesn't match versionName (0.5.9 beta).
So @toitoinebzh will you clarify that with upstream and please let us know when both issues are solved? For the JARs, we need the source (e.g. provided via a git submodules) – and for the version, versionName must be part of the tag name (so versionName should strip the " beta" (spaces in the versionName are a bad idea anyway)).

so to summarize : 
 * you need to provide JARs source to respect the [inclusion policy](https://f-droid.org/docs/Inclusion_Policy/?title=Inclusion_Policy)
> Binary dependencies such as JAR files have to be replaced by source-built versions or used from a trusted repository (see manual).

where can I find these sources ?
 * tag name should match versionName
